helm-chart/values.yaml

```
@@ -0,0 +1,130 @@
+
+postgresql:
+  enabled: false
+  externalUrl: ""
+  auth:
+    username: formbricks
+    password: formbrickspassword
+    database: formbricks
+  primary:
+    persistence:
+      enabled: true
+      size: 10Gi
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)